respa/settings.py

```diff
@@ -5,11 +5,32 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
 import environ
-import raven
+import sentry_sdk
+import subprocess
 from django.utils.translation import ugettext_lazy as _
 from django.core.exceptions import ImproperlyConfigured
+from sentry_sdk.integrations.django import DjangoIntegration
 
 
+def get_git_revision_hash():
```

Comment:
Alternative without the git executable dependency would be from the old sdk: https://github.com/getsentry/raven-python/blob/d7d14f61b7fb425bcb15512f659626648c494f98/raven/versioning.py#L11

Reply:
Another way without having to dig through the git repo would be to write the release name to a file during the execution of the release script. That would be faster than calling an external process during Django startup.

Reply:
After a bit of back and forth, agreement has reach that `git describe --tags --long --always` is actually quite informative. I'll also pick up the revision file as backup.